When checking if the architecture supports the %time register
instruction, do not force -mcpu to v9 while doing so. Otherwise it's
like "let's see if this v9 instruction exists when I force the compiler
to think I'm using v9", which is non-sensical.

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Dario: make the patch to be applied with fuzz factor 0]
Signed-off-by: Dario Binacchi <dario.binacchi@amarulasolutions.com>

diff -Naurp ./snort-2.9.11.1-orig/configure.in snort-2.9.11.1/configure.in
--- ./snort-2.9.11.1-orig/configure.in	2018-05-10 12:20:19.253510678 +0200
+++ snort-2.9.11.1/configure.in	2018-05-10 12:40:18.547584998 +0200
@@ -900,8 +900,6 @@
 
 # check for sparc %time register
 if eval "echo $host_cpu|grep -i sparc >/dev/null"; then
-    OLD_CFLAGS="$CFLAGS"
-    CFLAGS="$CFLAGS -mcpu=v9 "
     AC_MSG_CHECKING([for sparc %time register])
     AC_COMPILE_IFELSE(
     [AC_LANG_PROGRAM(
@@ -915,8 +913,6 @@
     AC_MSG_RESULT($sparcv9)
     if test "x$sparcv9" = "xyes"; then
         AC_DEFINE([SPARCV9],[1],[For sparc v9 with %time register])
-    else
-        CFLAGS="$OLD_CFLAGS"
     fi
 fi
 
